webpack/ForemanInventoryUpload/Components/TabHeader/TabHeader.js

```diff
@@ -1,26 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Grid, Button, Icon } from 'patternfly-react';
+import { Grid, Button, Icon, DropdownItem, Dropdown, DropdownToggle, DropdownToggleAction } from 'patternfly-react';
 import { noop } from 'foremanReact/common/helpers';
 import { sprintf, translate as __ } from 'foremanReact/common/I18n';
 import { isExitCodeLoading } from '../../ForemanInventoryHelpers';
 import './tabHeader.scss';
 
-const TabHeader = ({ exitCode, onRestart, onDownload, toggleFullScreen }) => (
+const onActionToggle = () => {
+  setIsActionOpen(prev => !prev);
```

Comment:
Where is this setter coming from?

I think it would be better to use React's `useState` inside the `TabHeader` component to trigger a component re-render

Reply:
I copied and pasted that in from the Katello Errata tab